scripts/test-travis.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+# =============================================================================
+
+set -e
+
+# If this is nightly, use tfjs-core at master.
+if [ "$TRAVIS_EVENT_TYPE" = cron ]
+then
```

Comment:
Please add a `echo` here stating that the test is running against tfjs-core HEAD.

Reply:
Good idea. Added